_dashboards/management/multi-data-sources.md

```diff
@@ -93,11 +92,13 @@ To select a data source through the Dev Tools console, follow these steps:
 6. Repeat the preceding steps for each data source you want to select.
 
 ### Upload saved objects to a dashboard from connected data sources
-To upload saved objects from connected data sources to a dashboard with multiple data sources, export them as an NDJSON file from the data source's **Saved object management** page. Then upload the file to the dashboard's **Saved object management** page. This method can make it easier to transfer saved objects between dashboards. The following 20-second video shows this feature in action.
 
+To upload saved objects from connected data sources to a dashboard with multiple data sources, export them as an NDJSON file from the data source's **Saved object management** page. Then upload the file to the dashboard's **Saved object management** page. This method can make it easier to transfer saved objects between dashboards. The following 20-second video shows this feature in action.
 
 <img src="{{site.url}}{{site.baseurl}}/images/dashboards/import_saved_objects_with_file_upload.gif" alt="Multiple data sources in Saved object management">{: .img-fluid}
 
+### Import saved objects from a connected data source
```

Comment:
`Import saved objects from a connected data source`, which is belong to section `Upload saved objects to a dashboard from connected data sources`, so it could be "####", not the same level of `Upload saved objects to a dashboard from connected data sources`